src/app/+display/section/section.component.html

```diff
@@ -1,15 +1,15 @@
 <div class="card-header font-weight-bold text-primary text-capitalize">
-  <span>{{section.name}}</span>
+  <span>{{ section.name }}</span>
   <div class="float-right" *ngIf="section.shared">
     <div class="embed-dropdown d-inline-block" placement="bottom-right" ngbDropdown>
-      <i class="fa fa-lg fa-share-alt" aria-hidden="true" ngbDropdownToggle></i>
+      <i class="fa fa-lg fa-share-alt" aria-hidden="true" ngbDropdownToggle [attr.aria-label]="'DISPLAY.EMBEDDABLE.SHARE' | translate"></i>
       <div class="dropdown-menu" ngbDropdownMenu>
         <div class="form-group col-12">
-          <label>{{ 'DISPLAY.EMBEDDABLE.LABEL' | translate }}</label>
+          <label for="embedCode">{{ "DISPLAY.EMBEDDABLE.LABEL" | translate }}</label>
           <div class="input-group">
-            <textarea class="form-control" type="text" rows="4" ng-bind-html [value]="getEmbedSnippet()" #embedCode></textarea>
+            <textarea class="form-control" type="text" rows="4" id="embedCode" ng-bind-html [value]="getEmbedSnippet()" #embedCode></textarea>
```

Comment:
I am concerned about using a generic `id` attribute of `embedCode` here.
I believe that if multiple sections are added, then we can have multiple ids.
We either need to generate something random to append, or use something distinct being passed to the section component and append that.